DNN Platform/Modules/DigitalAssets/View.ascx.cs

```diff
@@ -127,11 +127,11 @@ protected string InvalidCharactersErrorText
             }
         }
 
-        protected int MaxUploadSize
+        protected long MaxUploadSize
```

Comment:
This creates backward compatibility issue. Must deprecate and create another one.

Reply:
George, what issues do you expect within this module? there should be no 3rd party extension accessing this control.

Reply:
Who knows? There might be - we can't break compatibility assuming this!